src/components/Form/FormNumericInput.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+
+import _ from 'lodash'
+
+import { Field, FormikErrors, FormikTouched } from 'formik'
+import { Col, FormGroup, Row } from 'reactstrap'
+
+import FormLabel from './FormLabel'
+import NumericInput from './NumericInput'
+
+export interface FormNumericInputProps<T> {
+  identifier: string
+  label: string
+  help?: string | React.ReactNode
+  step?: number | string
+  precision?: number
+  errors?: FormikErrors<T>
+  touched?: FormikTouched<T>
+}
+
+export function FormNumericInput<T>({
+  identifier,
+  label,
+  help,
+  step,
+  precision,
+  errors,
+  touched,
+}: FormNumericInputProps<T>) {
+  const isTouched = _.get(touched, identifier)
+  const errorMessage = _.get(errors, identifier)
+  const showError = errorMessage && isTouched
+
+  return (
+    <Field>
+      {() => (
```

Comment:
Can this `<Field />` be removed? NumericInput does its own Formik `<Field />`

Reply:
Yeah `<Field />` doesn't belong here. Removed it.